mlflow/R/mlflow/NAMESPACE

```diff
@@ -61,7 +57,6 @@ export(mlflow_start_run)
 export(mlflow_ui)
 export(mlflow_uninstall)
 import(forge)
-import(rlang)
```

Comment:
Why are we reomoving rlang dependency? I am not 100% sure how are these supposed to work but we still use rlang::rlang::env_unbind(.globals, "run_params").

Reply:
The NAMESPACE file is autogenerated from Roxygen-style comments - when we removed the inlined crate source code, we also removed the sole Roxygen directive that imported rlang, so it disappeared from NAMESPACE. In particular it seems the only reason to have an `import(rlang)` in NAMESPACE is if we want to call `rlang` functions without using `::` ([see docs](http://r-pkgs.had.co.nz/namespace.html#imports))

We still do depend on rlang in our DESCRIPTION file's imports, so `rlang` will always be installed when our package is installed, as described in http://r-pkgs.had.co.nz/namespace.html#imports

Reply:
OK, thanks for the clarification!